src/dex/woo-fi-v2/woo-fi-v2-math.ts

```diff
@@ -46,6 +46,14 @@ export class WooFiV2Math {
   _querySellBase(baseToken: Address, baseAmounts: bigint[]): bigint[] | null {
     try {
       const _state = this.state.tokenStates[baseToken];
+
+      if (!_state.woFeasible) {
+        this.logger.error(
```

Comment:
Can we also log `this.state.oracleTimestamp` to ease debugging for such cases?

Reply:
done